tests/providers/google/cloud/hooks/test_dataflow.py

```diff
@@ -1163,7 +1163,7 @@ def test_dataflow_job_cancel_job(self):
         )
         # fmt: on
         mock_update.assert_called_once_with(
-            body={'requestedState': 'JOB_STATE_CANCELLED'},
```

Comment:
It does not look good, because you make a change in the string not constant Can you check it out?